### DISCUSSION TITLE
something went wrong while trying to lad main page

### DISCUSSION THREAD

**Discussion by the-wildguy** [ppy/osu#28331] [Q&A] [answered]
### Type

Game behaviour

### Bug description

loaded page while loading a map and signing in has an error

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/90672169/68f55c18-398a-48cd-8c29-bc6ec62565d1)

### Version

2024.521.2

### Logs

game crashed before i could get logs (i just get random crashes on linux)

**Comment by BlockHead007** [comment]
You can find logs in your homedir/.local/share/osu folder

**Comment by bdach** [comment]
As above, would need logs to say something definitive, but the screenshot in the OP just looks like general network failure to me (API in failing state, canceled API requests).

**Reply by the-wildguy** [reply]
If it helps it was after loading osu.ppy.sh it gave a too many requests error (this was just before loading the game)

**Reply by bdach** [reply]
Well it "helps" in the sense that the immediate cause is obvious (you got rate-limited). The follow up question is what were you doing *before* that that got you rate-limited. Were you mass-downloading beatmaps or something?

**Reply by the-wildguy** [reply]
Using the API but my API key broke for some reason so I was trying to generate a new one

**Reply by bdach** [reply, marked as answer]
Sounds likely that whatever you were doing with that API key that "broke" it rate-limited you too?

**Reply by the-wildguy** [reply]
thats what i think
